concourse/pipelines/deploy.yml

```diff
@@ -331,16 +331,22 @@ jobs:
 
   - name: smoke-test-frontend
     plan:
-    - get: govuk-infrastructure
-      resource: govuk-infrastructure-frontend
-      passed:
-      - deploy-frontend
-      trigger: true
-    - file: govuk-infrastructure/concourse/tasks/basic-smoke-test.yml
+    - in_parallel:
```

Comment:
does this `in_parallel` useful here, there seems to be only one resource being retrieved.

Reply:
Thanks, I've removed this.